### DISCUSSION TITLE
onnxruntime_test

### DISCUSSION THREAD

**Discussion by alicera** [microsoft/onnxruntime#7630] [Other Q&A]
how to use the command onnxruntime_test in linux.
I want to test the onnx

**Comment by georgen117** [comment]
I am making the assumption that you are asking how to run `onnxruntime_test_all` if I am wrong let me know and I will try and update the answer.

build onnxruntime using your prefered build options. 

example:
`./build.sh --config RelWithDebInfo --parallel  --build_shared_lib`

after the build is complete cd to the build output folder and run the `onnxruntime_test_all`

```
cd build/Linux/RelWithDebInfo
./onnxruntime_test_all
```

`onnxruntime_test_all` is built using the gtest framework so look up gtest command-line options to get more control over running the unit tests.
